src/Ewmh.cc

```diff
@@ -693,6 +698,27 @@ void Ewmh::setupClient(WinClient &winclient) {
 
 }
 
+void Ewmh::updateVisibleName(const std::string &new_title, Focusable &win) {
+    std::string netname = win.getTextProperty(m_net->wm_name);
+    if (netname != new_title) {
+         XTextProperty tp;
+	 /* Make a copy of the new title for const-correctness purposes */
+	 std::vector<char> ntbuf(new_title.begin(), new_title.end());
+	 ntbuf.push_back(0);
+	 char* ntbuf_cstr = &ntbuf[0];
```

Comment:
imho, `const char** nt = &new_title[0];` should be good to go.

Reply:
Unfortunately that leads to an "invalid conversion from 'const char**' to 'char**' error in the compiler that I don't see any other way around.